src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const RENTAL_COST_PER_DAY = 40;
+  const totalCOst = days * RENTAL_COST_PER_DAY;
```

Comment:
There's a typo in the variable name `totalCOst`. Variable names should be in camelCase and spelled correctly. Consider renaming it to `totalCost`.